src/components/Card.js

```diff
@@ -41,11 +66,12 @@ export default class Card {
     this._cardElement = this._getElement();
     this._cardImage = this._cardElement.querySelector(".card__image");
     this._cardTitle = this._cardElement.querySelector(".card__description");
-
+    this.cardLikeCount = this._cardElement.querySelector(".card__like-counter");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**